app/Services/AccountService.php

```diff
@@ -2,16 +2,14 @@
 
 namespace App\Services;
 
+use App\Http\Controllers\AccountController;
 
 
-class AccountService {
-
+class AccountService
+{
     public function getWishlistByUserId($id) {
-        //more code goes here
+        $ac = new AccountController;
```

Comment:
not sure what's happening here. you should be adding a constructor to AccountController that takes a private AccountService parameter. As far as our dependency injection setup goes, in general controllers will depend on services.

Reply:
this function should return an array of games

Reply:
Fixed issues